lib/db.js

```diff
@@ -18,147 +15,70 @@ Returns true if DB is contacted successfully within 10 secs and has at least 1 m
 */
 module.exports.getDBStatus = async () => {
     try {
-        const lastMigration = await knex('knex_migrations').select()
-            .orderBy('id', 'desc')
+        const lastMigration = await knex("knex_migrations").select()
+            .orderBy("id", "desc")
             .limit(1)
             .timeout(10 * 1000, {cancel: true})
         if (lastMigration.length === 0) {
-            console.error('error with DB : no migrations')
+            console.error("error with DB : no migrations")
             return false
         }
         return true
     } catch(err) {
-        console.error('status check got error with DB', err)
+        console.error("status check got error with DB", err)
         return false
     }
 }
 
-module.exports.getPhoneNumberList = async () => {
```

Comment:
on pourrait carrément supprimer la table phoneNumbers dans une migration ? 
Ah non parce qu'on perdrait des données de conf... Ok donc non.

Reply:
oui, je suppose que c'est mieux de garder les données existantes, donc pas de migration de la table stats et phoneNumbers dans cette itération.